src/Exceptions/RateLimitExceededException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Laravel\Forge\Exceptions;
+
+use Exception;
+
+class RateLimitExceededException extends Exception
+{
+    /**
+     * The timestamp that the rate limit will be reset.
+     *
+     * @var int|null
+     */
+    public $rateLimitResetsAt;
+
+    /**
+     * Create a new exception instance.
+     *
+     * @return void
+     */
+    public function __construct($rateLimitReset)
+    {
+        parent::__construct('Too Many Attempts.');
```

Comment:
Looking through the Forge codebase, we should probably use a similar message:

> Too many requests.

Reply:
Hi James,

I originally used `Too many requests.` however when I checked the response returned from the Forge API the actual response is:

`{
	"message": "Too Many Attempts."
}`